### ISSUE TITLE
Feature to automatically adjust slowmode

### ISSUE THREAD

**Issue by ItsAzaria** [the-programmers-hangout/Chronos#9]
This should take into account the config for slowmode thresholds in the config and use a timer, or perhaps a clever implementation using some kind of EvictingQueue and looking at the messages creation date

**Comment by ayezinzu** [comment]
May I work on this issue ?

**Comment by ItsAzaria** [comment]
Yes! Please do. I believe there may be someone else who's started it, double check on discord. You're welcome to work on it together too
